app/models/nic/bond.rb

```diff
@@ -38,12 +19,8 @@ def self.humanized_name
 
     private
 
-    def ensure_virtual
-      self.virtual = true
-    end
-
     def enc_attributes
-      @enc_attributes ||= (super + %w(mode attached_devices bond_options))
+      @enc_attributes ||= (super + %w(mode bond_options))
```

Comment:
[alias_method_chain](http://apidock.com/rails/Module/alias_method_chain) would have to be used instead

this should work (note that we can't use memoization, since it would be already set by without feature method)
```ruby
alias_method_chain :enc_attributes, :attached_devices

def enc_attributes_with_attached_devices
  @enc_attributes = enc_attributes_without_attached_devices + ['attached_devices']
end
```